chromadb/api/local.py

```diff
@@ -239,10 +240,17 @@ def _query(
             ids = []
             metadatas = []
             # Remove plural from include since db columns are singular
-            db_columns = [column[:-1] for column in include if column != "distances"] + ["id"]
+            db_columns = (
+                [column[:-1] for column in include if column != "distances"] + ["id"] + ["uuid"]
+            )
             column_index = {column_name: index for index, column_name in enumerate(db_columns)}
             db_result = self._db.get_by_ids(uuids[i], columns=db_columns)
 
+            # sort db results by the order of the uuids
+            db_result = sorted(
```

Comment:
Should we do this sort in the db query directly?

Reply:
I think that would be better - I wasnt sure how to do that.

Reply:
SORT BY in the SQL should work right?

Reply:
Oh true never mind. I think thats not possible here. Sorry - misread.

Reply:
There might be some workaround but I'm not sure with duckdb / clickhouse specifically.